src/foam/nanos/dig/DigUtil.java

```diff
@@ -0,0 +1,84 @@
+/**
+ * @license
+ * Copyright 2019 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package foam.nanos.dig;
+
+import foam.core.X;
+import foam.lib.*;
+import foam.lib.csv.CSVOutputter;
+import foam.lib.json.JSONParser;
+import foam.lib.json.Outputter;
+import foam.lib.json.OutputterMode;
+import foam.nanos.dig.exception.DigErrorMessage;
+import foam.nanos.http.Format;
+
+import javax.servlet.http.HttpServletResponse;
+import java.io.PrintWriter;
+
+public class DigUtil {
```

Comment:
If the only thing this class does is output exceptions, then wouldn't a more specific name be better? Something like DigExceptionOutputter?

Reply:
We plan to use it as a container for Dig related static utility methods.